dvclive/utils.py

```diff
@@ -73,3 +73,32 @@ def env2bool(var, undefined=False):
     if var is None:
         return undefined
     return bool(re.search("1|y|yes|true", var, flags=re.I))
+
+
+def standardize_metric_name(metric_name: str, framework: str) -> str:
+    """Map framework-specific format to DVCLive standard.
+
+    Use `{split}/` as prefix in order to seperate by subfolders.
+    Use `{train|eval}` as split name.
```

Comment:
Changing separators from `_` to `/` seems like a pretty straightforward and transparent change. Trying to standardize split names across all frameworks might be a bit too opinionated and unexpected. Is there some scenario where the metrics are a mess if we don't do this standardization?

Reply:
> Trying to standardize split names across all frameworks might be a bit too opinionated and unexpected. 
I just saw the opportunity to apply some friendly tyranny 😅 after seeing that many other ML loggers don't respect the original name passed by the framework.

> Is there some scenario where the metrics are a mess if we don't do this standardization?

Not really although for some (i.e. pytorch lightning) we do need to apply some "unexpected" refactor because of the `_{epoch|step}` suffix

Reply:
> I just saw the opportunity to apply some friendly tyranny 😅 after seeing that many other ML loggers don't respect the original name passed by the framework.

Ah, I didn't realize that was the case.

I guess replacing all `_` with `/` will be too aggressive, even if documented, so maybe we are stuck with some decisions on when to split. I'm still not sure we should automatically rename `valid` to `eval` or add `train` prefixes, for example. If you feel strongly about it, though, I don't think it's worth blocking the PR over.